pkg/argocd/argocd.go

```diff
@@ -187,6 +187,10 @@ func FilterApplicationsForUpdate(apps []v1alpha1.Application, patterns []string)
 	return appsForUpdate, nil
 }
 
+func parseImageList(updateImage string) []string {
+	return strings.Split(updateImage, ",")
+}
```

Comment:
I like that this becomes externalized in its own method. Maybe it would make sense to also pull in the white-space trimming into this method, what do you think? Also, I think it could be worth to spend this method its own unit test :)

Reply:
Once I pulled in the white space, I noticed the next step was to convert to a ContainerImage so I pulled that in as well. I just focused tests on the string parsing since NewFromIdentifier seems well covered.